sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -894,4 +894,30 @@ class DataFrameAggregateSuite extends QueryTest with SharedSQLContext {
         error.message.contains("function min_by does not support ordering on type map<int,string>"))
     }
   }
+
+  test("SPARK-27425: count_if function") {
```

Comment:
In general, we don't use SPARK JIRA id for new feature test case name. Could you remove `SPARK-27425: `?